packages/react-app/src/components/giveaways/cards/Card.tsx

```diff
@@ -136,7 +146,7 @@ const GiveawayCard = ({
             <span role="img" aria-label="party emoji">
               🥳
             </span>{' '}
-            {giveaway.winners.length > 0 ? getWinnerStr() : 'Pending'}
+            {giveaway.winners.length > 0 ? getWinnerStr() : 'No Winners'}
```

Comment:
Maybe it's just better to rollback to pending? Still provides a better explanation 👍

Reply:
Maybe we should differentiate when there are pending winners and invalid giveaways, otherwise it will display `Pending` for both scenarios. We could show `Pending` for the first scenario, and `No Winners` for the second